src/Modules/CalcOffence.lua

```diff
@@ -3729,6 +3734,10 @@ function calcs.offence(env, actor, activeSkill)
 			env.player.output = newEnv.player.output
 
 			-- Make any necessary corrections to output
+			-- Don't show attack rate for non-channeled skills
+			if not usedSkill.skillTypes[SkillType.Channelled] then
+				env.player.output.Speed = nil
```

Comment:
I don't think you can set it to `nil`. It will break many breakdown print outs for various GC combos:
```
				breakdown.Speed = {
					"Both weapons:",
					s_format("(%.2f + %.2f) / 2", output.MainHand.Speed, output.OffHand.Speed),
					s_format("= %.2f", output.Speed),
				}
```
Set it to 0 instead.
```suggestion
				env.player.output.Speed = 0
```

Reply:
Setting it to 0 unfortunately displays it

![image](https://user-images.githubusercontent.com/67038113/114193820-2e1c3a00-994f-11eb-9a16-42dc5b7ed6e1.png)

I might just strip this line, this'll display an attack rate of 1 (from the time override) and be accurate with the displayed DPS anyway.

Reply:
you can also modify the display to not show it if it's 0 using a conditional function